storage/table.go

```diff
@@ -127,3 +143,109 @@ func (c *TableServiceClient) DeleteTable(table AzureTable) error {
 	}
 	return nil
 }
+
+// SetTablePermissions sets up table ACL permissinos.
+func (c *TableServiceClient) SetTablePermissions(table AzureTable, accessPolicy TableAccessPolicy, timeout int) (err error) {
+	params := url.Values{
+		"comp": {"acl"},
+	}
+
+	if timeout > 0 {
+		params.Add("timeout", strconv.Itoa(timeout))
+	}
+
+	uri := c.client.getEndpoint(tableServiceName, string(table), params)
+	headers := c.client.getStandardHeaders()
+
+	var permissions = generateTablePermissions(accessPolicy)
+
+	// generate the XML for the SharedAccessSignature if required.
+	accessPolicyXML, err := generateAccessPolicy(accessPolicy.ID, accessPolicy.StartTime,
+		accessPolicy.ExpiryTime,
+		permissions)
+	if err != nil {
+		return err
+	}
+
+	var resp *odataResponse
+
+	// temp logging to figure out why dying in travis
+	log.Printf("access policy %s", accessPolicyXML)
+
+	if accessPolicyXML != "" {
+		headers["Content-Length"] = strconv.Itoa(len(accessPolicyXML))
+		resp, err = c.client.execTable("PUT", uri, headers, strings.NewReader(accessPolicyXML))
+	} else {
+		resp, err = c.client.execTable("PUT", uri, headers, nil)
+	}
+
+	if err != nil {
+		return err
+	}
+
+	if resp != nil {
+		defer func() {
+			closeErr := resp.body.Close()
+			if closeErr != nil && err == nil {
+				err = closeErr
+			}
+		}()
+
+		if resp.statusCode != http.StatusNoContent {
+			err = errors.New("Unable to set permissions")
+			return err
+		}
+	}
+	return nil
+}
+
+// GetTablePermissions gets the table ACL permissions
```

Comment:
Here it is the other documentation link
https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/get-table-acl